koku/cost_models/serializers.py

```diff
@@ -54,6 +55,28 @@ class MarkupSerializer(serializers.Serializer):
     unit = serializers.ChoiceField(choices=MARKUP_CHOICES, required=False)
 
 
+class DistributionSerializer(BaseSerializer):
+    """Serializer for distribution options"""
+
+    DISTRIBUTION_OPTIONS = {"distribution_type", "worker_cost", "platform_cost"}
+
+    distribution_type = serializers.ChoiceField(choices=metric_constants.DISTRIBUTION_CHOICES, required=False)
+    platform_cost = serializers.BooleanField(required=False)
+    worker_cost = serializers.BooleanField(required=False)
+
+    def validate(self, data):
+        """Run validation for distribution options."""
+
+        diff = self.DISTRIBUTION_OPTIONS.difference(data)
+        if diff == self.DISTRIBUTION_OPTIONS:
+            return {"distribution_type": metric_constants.CPU_DISTRIBUTION, "platform_cost": True, "worker_cost": True}
```

Comment:
Let's include a test that hits this line. It seems we need a test that sends a `distribution_info: {}` json.